lib/td/updater.rb

```diff
@@ -355,6 +344,17 @@ def jar_update(hourly = false)
         if status
           $stdout.puts "Installed td-import.jar v#{version} in '#{Updater.jarfile_dest_path}'.\n"
           File.rename 'td-import.jar.new', 'td-import.jar'
+
+          File.open('VERSION', 'w') {|f|
+            if hourly
+              f.print "#{version} via hourly jar auto-update"
+            else
+              f.print "#{version} via import:jar_update command"
+            end
+          }
+          File.open('td-import-java.version', 'w') {|f|
+            f.print "#{version} #{updated}"
+          }
         else
           $stdout.puts "Update of td-import.jar failed." unless ENV['TD_TOOLBELT_DEBUG'].nil?
```

Comment:
This is a comment related to this fix. But I think that the above message should be shown with any log level. I mean that it's better to remove `unless ENV['TD_TOOLBELT_DEBUG'].nil?` condition. And we can put some messages here to avoid users' confusion.

```ruby
$stdout.puts "Update of td-import.jar failed."
$stdout.puts "Please execute 'td import:jar_update' later to update td-import.jar correctly."
```